src/theming/sap_fiori_3.scss

```diff
@@ -31,4 +31,19 @@
   --fdWizard_Content_Transparent_Background: transparent;
   --fdWizard_Selection_Bar_Height: 0.188rem;
   --fdWizard_Selection_Bar_Background: var(--sapSelectedColor);
+
+  /* Slider */
+
+  --fdSlider_Track_Height: 0.25rem;
+  --fdSlider_Track_BorderRadius: 0.25rem;
+  --fdSlider_Tick_Height: 1rem;
+  --fdSlider_Track_Background: var(--sapField_BorderColor);
+  --fdSlider_Track_Active_Background: var(--sapActiveColor);
+  --fdSlider_Track_Active_Border: var(--sapActiveColor);
+  --fdSlider_Handle_Background: var(--sapButton_Background);
+  --fdSlider_Handle_Border_Color: var(--sapField_BorderColor);
+  --fdSlider_Handle_Background_Hover: var(--sapButton_Hover_Background);
+  --fdSlider_Handle_Border_Color_Hover: var(--sapButton_Hover_BorderColor);
+  --fdSlider_Handle_Background_Active: var(--sapButton_Active_Background);
+  --fdSlider_Handle_Border_Color_Active: var(--sapButton_Active_BorderColor);
```

Comment:
Custom css variables are meant for situations where high contrast has a different value than light/dark and sap-theming repo has not defined a variable for it. A lot of these seem to be the same in all themes? If so, just use the `var(--sap*)` variable in `slider.scss`

For example, ` fdSlider_Track_Background`, `fdSlider_Handle_Background`, `fdSlider_Handle_Border_Color`, `fdSlider_Handle_Background_Hover`, `fdSlider_Handle_Border_Color_Hover`, `fdSlider_Handle_Background_Active`, `fdSlider_Handle_Border_Color_Active` should be removed and just use the sap variable in their place in the code. 

The goal is to keep these theming files as small as possible as we slowly work these values back into sap theming repo - we ideally want to have 0 custom css variables in this repository.

Reply:
thanks! I removed variables that are the same for all themes